client/my-sites/google-my-business/stats/index.js

```diff
@@ -0,0 +1,159 @@
+import SectionHeader from 'components/section-header';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import StatsNavigation from 'blocks/stats-navigation';
+import { getSelectedSiteSlug, getSelectedSiteId } from 'state/ui/selectors';
+
+class GoogleMyBusinessStats extends Component {
+	static propTypes = {
+		actionData: PropTypes.array.isRequired,
+		locationData: LocationType.isRequired,
+		searchData: PropTypes.arrayOf( SearchDataType ).isRequired,
+		searchDataTotal: PropTypes.number.isRequired,
+		siteId: PropTypes.number.isRequired,
+		siteSlug: PropTypes.string.isRequired,
+		translate: PropTypes.func.isRequired,
+		viewData: PropTypes.array.isRequired,
+	};
+
+	render() {
+		const {
+			actionData,
+			locationData,
+			searchData,
+			searchDataTotal,
+			siteId,
+			siteSlug,
+			translate,
+			viewData,
+		} = this.props;
+
+		return (
+			<Main wideLayout>
+				<DocumentHead title={ translate( 'Stats' ) } />
+				<SidebarNavigation />
+				<StatsNavigation selectedItem={ 'googleMyBusiness' } siteId={ siteId } slug={ siteSlug } />
+
+				<div>
+					<Card className="gmb-location">
+						<Location location={ locationData } />
+						<Button className="gmb-location__button">
+							{ translate( 'Update Listing' ) } { <Gridicon icon={ 'external' } /> }
+						</Button>
+					</Card>
+
+					<SectionHeader label={ translate( 'How customers search for your business' ) } />
+					<Card>
+						<PieChart
+							data={ searchData }
+							title={ translate( '%(searchDataTotal)d Total Searches', {
+								args: { searchDataTotal },
+							} ) }
+						/>
+					</Card>
+
+					<SectionHeader
+						label={ translate( 'Where your customers view your business on Google' ) }
+					/>
+					<Card>
+						<CardHeading tagName={ 'h2' } size={ 16 }>
+							{ translate( 'The Google services that customers use to find your business' ) }
+						</CardHeading>
+						<Chart data={ viewData } />
+					</Card>
+
+					<SectionHeader label={ translate( 'Customer Actions' ) } />
+					<Card>
+						<CardHeading tagName={ 'h2' } size={ 16 }>
+							{ translate( 'The most common actions that customers take on your listing' ) }
+						</CardHeading>
+						<Chart data={ actionData } />
+					</Card>
+				</div>
+			</Main>
+		);
+	}
+}
+
+const dataSummer = ( prevResult, datum ) => prevResult + datum.value;
+
+const searchData = [
+	{
+		name: translate( 'Direct' ),
+		description: translate(
+			'Customers who find your listing searching for you business name or address'
+		),
+		value: random( 300, 500 ),
+	},
+	{
+		name: translate( 'Discovery' ),
+		description: translate(
+			'Customers who find your listing searching for a category, product, or service'
+		),
+		value: random( 200, 400 ),
+	},
+];
+const searchDataTotal = searchData.reduce( dataSummer, 0 );
+
+const viewData = range( 19, 30 ).map( day => ( {
+	value: random( 10, 90 ),
+	nestedValue: random( 5, 80 ),
+	label: translate( 'Mar %(day)d', {
```

Comment:
Please don't wrap text in `translate()` calls that is not meant to be translated, as we'll be sending this for translation automatically.